commands/make/make.drush.inc

```diff
@@ -649,7 +649,7 @@ function drush_make_convert_make_to_composer($info) {
     switch ($project['type']) {
       case 'core':
         $project['name'] = $core_project_name;
-        $projects[$project['name']] = '^' . str_replace('x', '*', $project['version']);
+        $projects[$project['name']] = str_replace('x', '*', $project['version']);
```

Comment:
Versions will not always contain `.x`, so this change makes some versions that were previously converted to `^1.2.3` to be converted to `1.2.3` (pinned to a specific version). The old line wasn't quite right either, as it might produce `^1.*` instead of `1.*`. A correct transformation should handle both of these cases.